src/components/Multiselect/Multiselect.vue

```diff
@@ -20,15 +20,13 @@
         <Badge
           v-for="option in modelValue"
           :key="option.label || option"
-          variant="turquoise"
           :size="size"
-          class="my-2 ml-2"
         >
           <div class="flex items-center">
             {{ option.label || option }}
             <LobButton
               variant="none"
-              class="mt-0.5 ml-1"
+              class="!pr-0 !pl-1"
```

Comment:
try adding `h-min` here I think it'll look better

Reply:
Yep, you were right.